docs/README.md

```diff
@@ -124,12 +124,12 @@ must authorize the Azure Application you provision (with [provided terraform mod
-| Mail                                                                                                   | [data](https://github.com/Worklytics/psoxy/tree/main/docs/sources/microsoft-365/outlook-mail/example-api-responses) - [rules](https://github.com/Worklytics/psoxy/tree/main/docs/sources/microsoft-365/outlook-mail/outlook-mail.yaml) | `User.Read.All` `Group.Read.All`  `Mail.ReadBasic.All` `MailboxSettings.Read`                                                                     |
-| Teams (**__beta__**)                                                                                   | [data](https://github.com/Worklytics/psoxy/tree/main/docs/sources/microsoft-365/msft-teams/example-api-responses) - [rules](https://github.com/Worklytics/psoxy/tree/main/docs/sources/microsoft-365/msft-teams/msft-teams.yaml)| `User.Read.All` `Team.ReadBasic.All` `Channel.ReadBasic.All` `Chat.Read.All` `ChannelMessage.Read.All` `CallRecords.Read.All` `OnlineMeetings.Read.All`  |
+| Source&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Examples &nbsp;&nbsp;                                                                                                        | Application Scopes                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+|--------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
OK, not for this one; but... I'd favor regular HTML syntax for tables over Markdown's extended one (not included in the original Markdown spec). I feel this is ugly and hard to edit (unless you're using some magic IntelliJ formatter?).

Reply:
fair ... does that work well?

Reply:
so I can do it, if I drop wrapping the oauth scope tokens with backticks

https://github.com/Worklytics/psoxy/blob/186814436d9be761e6f96a388303a52eed7a4c15/docs/README.md#microsoft-365

if I wrap with back ticks, looks like garbage:
https://github.com/Worklytics/psoxy/blob/c82a9a0c1d1b116fecd66b1a73e7a8d9a601d95c/docs/README.md#microsoft-365

we use the backtick-style formatting most in the other data source documentation:

![SCR-20240508-igoy](https://github.com/Worklytics/psoxy/assets/102831/601be1e5-f06b-4ff4-b08a-ba0952b4b382)